drivers/soc/bl702/std/src/bl702_ef_cfg.c

```diff
@@ -85,11 +86,9 @@ static uint8_t bdiv, hdiv;
 void  ATTR_TCM_SECTION bflb_efuse_switch_cpu_clock_save(void)
 {
     /* all API should be place at tcm section */
-    bdiv = GLB_Get_BCLK_Div();
-    hdiv = GLB_Get_HCLK_Div();
-    rtClk = GLB_Get_Root_CLK_Sel();
+    hbnXclkClk = HBN_Get_XCLK_CLK_Sel();
+    hbnRootClk = HBN_Get_ROOT_CLK_Sel();
     HBN_Set_ROOT_CLK_Sel(HBN_ROOT_CLK_RC32M);
-    GLB_Set_System_CLK_Div(0, 0);
```

Comment:
cpu clock need restore to 32M.

Reply:
CPU clock is save from GLB function, but restored to HBN and this is different clocks. In my case this leads to using 32M RC clock after calling bflb_efuse_switch_cpu_clock_restore() and this breaks BLE RF interaction, cause it needs very stable clock from 32M external XTAL. Check this condition and you will see.